RELEASE-NOTES.md

```diff
@@ -1,5 +1,129 @@
+
+```bash
+❯ prefect work-pool provision-infra my-work-pool
+? Please select which Azure subscription to use: [Use arrows to move; enter to select]
+┏━━━━┳━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+┃    ┃ Name                 ┃ Subscription ID                      ┃
+┡━━━━╇━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
+│    │ Engineering          │ 13d                                  │
+│ >  │ Azure subscription 1 │ 6h4                                  │
+└────┴──────────────────────┴──────────────────────────────────────┘
+╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+│ Provisioning infrastructure for your work pool my-work-pool will require:                                      │
+│                                                                                                                │
+│     Updates in subscription Azure subscription 1                                                               │
+│                                                                                                                │
+│         - Create a resource group in location eastus                                                           │
+│         - Create an app registration in Azure AD prefect-aci-push-pool-app                                     │
+│         - Create/use a service principal for app registration                                                  │
+│         - Generate a secret for app registration                                                               │
+│         - Assign Contributor role to service account                                                           │
+│         - Create Azure Container Instance 'aci-push-pool-container' in resource group prefect-aci-push-pool-rg │
+│                                                                                                                │
+│     Updates in Prefect workspace                                                                               │
+│                                                                                                                │
+│         - Create Azure Container Instance credentials block aci-push-pool-credentials                          │
+│                                                                                                                │
+╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+Proceed with infrastructure provisioning? [y/n]: y
+...
+```
+
+This PR bolsters support for efficient work pool management across diverse cloud environments, delivering a tool for seamless infrastructure setup.
+
+See the following pull request for implementation details:
+- https://github.com/PrefectHQ/prefect/pull/11341
+
+### Enhancements
+- Added a `suspend_flow_run` method to suspend a flow run. — https://github.com/PrefectHQ/prefect/pull/11291
```

Comment:
```suggestion
- Added a `suspend_flow_run` method to suspend a flow run — https://github.com/PrefectHQ/prefect/pull/11291
```

Reply:
Some of these comments were dropped on my latest push, but I followed your suggestions 👍🏼